### PR TITLE
Impl FAQ low-fi

### PR DESCRIPTION
**Pull request by kozr**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Made according to #6 

I made a few design or naming decisions here. Would be nice to have them evaluated. e.g. index.js and lib

I also fixed styled components and bulma setup bugs.

Implements #25 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Works on staging site

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by danielshmidov — COMMENTED** [review]
Went Over Many things and tested the functionality of the website. Good work so far!

I would recommend waiting for another PR review before merging

File: src/pages/index.js
Comment (yehee) [inline src/pages/index.js:39]: Just out of curiosity, what would be the reason to make such a change?

Comment (yehee) [reply src/pages/index.js:39]: Hmm never mind! I guess it's just the way it got passed in.

Comment (kozr) [reply src/pages/index.js:39]: The problem with FeatureFlags is that CMS would have problems LOL. So unless we update cms, this is it. What's your opinion?

File: buildconfig.js
Comment (ianmah) [inline buildconfig.js:1]: what is this file for?

**Comment by ianmah** [comment]
How are we differentiating between `.js` and `.jsx`?

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (ianmah) [inline src/components/faqTemplates/ExpandableWithCategories.jsx:67]: Is there ever a case where we would need to add new categories? Not sure if CMS supports it but maybe we would want to make this more general to allow for any category?

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (ianmah) [inline src/components/faqTemplates/ExpandableWithCategories.jsx:72]: Could we make this more component like so usage looks more like `<Faq list={xxx} />` instead of `renderFaq(xxx)`

File: buildconfig.js
Comment (kozr) [reply buildconfig.js:1]: It's a hack for database schema that we're keeping so we can change variables while doing dev work. It'll also make it easy for us to implement the schema in production. Uncomment and run the `fireDb.updateConfig` function to update it. Do you have suggestions to do this neater?

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (ianmah) [inline src/components/faqTemplates/ExpandableWithCategories.jsx:7]: shall we use `p`?
```suggestion
  display: ${p=> (p.shouldDisplay ? 'block' : 'none')};
```

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (ianmah) [inline src/components/faqTemplates/ExpandableWithCategories.jsx:23]: is this needed

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (ianmah) [inline src/components/faqTemplates/ExpandableWithCategories.jsx:35]:
```suggestion
  return (
    <Accordion onClick={() => toggleExpansion(!isExpanded)}>
      <FaqQuestionContainer>{question}</FaqQuestionContainer>
      <FaqAnswerContainer shouldDisplay={isExpanded}>{answer}</FaqAnswerContainer>
    </Accordion>
  )
```

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:7]: > How are we differentiating between `.js` and `.jsx`?

jsx files will indicate React components and js will indicate files without React components.

Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:7]: that's a good idea. `props -> p`

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:23]: I'm leaving this so I could implement styles when working on High Fi

File: buildconfig.js
Comment (ianmah) [reply buildconfig.js:1]: Is the data in this file (ie `80%`) pulled in locally or is it example data? Is the schema enforced somehow?

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:65]: I don't recall any instances where logs/design made more faq categories. When they do, it'll probably be permanent from that point on so we can just make a commit to this file then

File: buildconfig.js
Comment (kozr) [reply buildconfig.js:1]: When the `fireDb.updateConfig` in index.js and the respective data in buildconfig.js are uncommented. The data would get pushed to firebase. So the data in this file are staging data. What do you mean by if the schema is enforced?

Comment (kozr) [reply buildconfig.js:1]: The data is pushed to Firebase and getStaticProps pulls data from Firebase right after.

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:70]: That sounds good, should we refactor common components in each section to src/lib/<section> so other templates could use it as well? e.g. under src/lib/Faq/Faq.jsx?

**Comment by kozr** [comment]
@ianmah One thing I missed when coming to the external dev team is your PR review. I finally feel at home.

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:7]: Are you implying only the styled-components should change `props -> p` or both react components and styled-components should change `props ->`

Comment (ianmah) [reply src/components/faqTemplates/ExpandableWithCategories.jsx:7]: mainly styled components I think, react components up to you but I would spread the props if there aren't that many props

File: buildconfig.js
Comment (ianmah) [reply buildconfig.js:1]: Ah I see thanks, was wondering if the schema forces future data to be in a certain format/hierarchy. Guess not

File: src/components/faqTemplates/ExpandableWithCategories.jsx
Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx]: I'm choosing not to do that because I'd also have to export ExpandableFaq as well and not every template uses that.

Comment (kozr) [reply src/components/faqTemplates/ExpandableWithCategories.jsx]: It's also very little code

File: src/lib/Containers.jsx
Comment (ianmah) [inline src/lib/Containers.jsx:11]: should we use constants for our breakpoints

File: src/lib/Containers.jsx
Comment (ianmah) [inline src/lib/Containers.jsx:19]: maybe 
```suggestion
  margin: 0 2vw;
```

**Review by ianmah — APPROVED** [review]

**Comment by kozr** [comment]
Thank you @ianmah

**Pull request merged**
